src/services/OutbreakService/OutbreakService.ts

```diff
@@ -363,3 +364,9 @@ export class OutbreakService {
     return periodsToFetch;
   };
 }
+
+export const isDiagnosed = (status: string) => {
+  if (status === ExposureStatusType.Diagnosed) {
+    return true;
+  }
```

Comment:
For good measure lets return false vs no value here and add a Boolean return type to the definition